great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -51,7 +51,8 @@ class OnboardingDataAssistant(DataAssistant):
             that is suspected to have quality issues may benefit from "flag_outliers".
         - include_column_names (list): A list containing the column names you wish to include.
         - exclude_column_names (list): A list containing the column names you with to exclude.
-        - include_column_name_suffixes (list): A list containing the column name suffixes you with to include.
+        - include_column_name_suffixes (list): A list containing the column name suffixes you wish to include.
```

Comment:
This will work only if the method `context.assistants.<your_data_assistant_name_here>.run` will return these arguments as available.